### PR TITLE
feat: add Search Input logic #1730

### PR DESCRIPTION
**Pull request by marcelina-hasiak**

### CODE REVIEW

**Comment by rast-mk1** [comment]
Looks like a whole bunch of less popular movies don't have backdrops. Maybe it would be better to use posters in search result grid instead?
Lack of backdrops will be a problem for `MovieHero`, but, it could have some default image if backdrop is not there.

File: app/components/UI/MovieGrid/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import MovieCard from '../MovieCard';
+import { StyledMovieGrid } from './MovieGrid.styles';
+
+export default function MovieGrid({ listOfMovies = [] }) {
+  return (
+    <StyledMovieGrid>
+      {listOfMovies.map(({ id, title, images }) => (
+        <MovieCard ratio="2/1" key={id} title={title} src={images?.backdrop?.l} />
+      ))}
+    </StyledMovieGrid>
```
Comment (rast-mk1) [inline app/components/UI/MovieGrid/index.js:11]: `MovieCard` should be using `id` to build `href` to specific movie page.
obraz

**Review by rast-mk1 — CHANGES_REQUESTED** [review]

**Review by weronika299 — COMMENTED** [review]
Could you pull the latest develop? Now you have changes from HomePage as well. If the PR includes only your changes, it will be more transparent.

**Comment by marcelina-hasiak** [comment]
I'm closing this PR, please visit: https://github.com/intive/patronage22_poznan_js/pull/103 to continue review.

**Comment by marcelina-hasiak** [comment]
I'm closing this PR, please visit: https://github.com/intive/patronage22_poznan_js/pull/103 to continue review.

**Pull request closed without merging**